controllers/namespace/exampleExclusionCMap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: namespace-exclusions
+  namespace: skiperator-system
+data:
+  istio-system: exclude
+  kube-node-lease: exclude
+  kube-public: exclude
+  kube-system: exclude
+  config-management-system: exclude
+  config-management-monitoring: exclude
+  asm-system: exclude
+  anthos-identity-service: exclude
+  binauthz-system: exclude
+  cert-manager: exclude
+  gatekeeper-system: exclude
+  gke-connect: exclude
+  gke-system: exclude
+  gke-managed-metrics-server: exclude
+  resource-group-system: exclude
+  kasten-io: exclude
+  vault: exclude
+  sysdig-agent: exclude
+  sysdig-admission-controller: exclude
+  instana-agent: exclude
+  kubecost: exclude
+  argocd: exclude
+  crossplane-system: exclude
+  upbound-system: exclude
+  instana-autotrace-webhook: exclude
+  fluentd: exclude
```

Comment:
her hadde man strengt tatt kun trengt å inkludere en nøkkel og ikke bry seg om verdien, men om man gjør det på følgende måte så blir det kanskje vanskeligere å gjøre feil. vet ikke om det er hensiktsmessig.

Reply:
Seems it has to be a string-to-string map. The only alternative would be to put an array in a string and decode it as JSON in code, but that has its own complexity, so I guess this is fine